mind_matter_api/api/emergency_contacts.py

```
@@ -0,0 +1,98 @@
+from flask import jsonify, request, current_app
+from marshmallow import ValidationError
+from mind_matter_api.schemas import EmergencyContactSchema
+from mind_matter_api.services.emergency_contacts import EmergencyContactsService
+from mind_matter_api.utils.decorators import require_auth, require_owner
```

Ruff (0.8.2)

`mind_matter_api.utils.decorators.require_owner` imported but unused

Remove unused import: `mind_matter_api.utils.decorators.require_owner`

(F401)